tests/ops/integration_tests/saas/test_appsflyer_task.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+# import pdb -- used for trouble shooting
+
+from fides.api.models.policy import Policy
+from tests.ops.integration_tests.saas.connector_runner import ConnectorRunner
+
+
+@pytest.mark.integration_saas
+class TestAppsflyerConnector:
+    def test_connection(self, appsflyer_runner: ConnectorRunner):
+        appsflyer_runner.test_connection()
+
+    async def test_access_request(
+        self, appsflyer_runner: ConnectorRunner, policy, appsflyer_identity_email: str
+    ):
+        access_results = await appsflyer_runner.access_request(
+            access_policy=policy, identities={"email": appsflyer_identity_email}
+        )
```

Comment:
add an assertion here to verify that the user we return matches the identity email